src/Data/Abstract/AccessControls/Instances.hs

```diff
@@ -302,40 +301,6 @@ instance AccessControls1 Haskell.Instance
 instance AccessControls1 Haskell.KindTupleType
 instance AccessControls1 Haskell.FunctionalDependency
 
-
-instance AccessControls1 Java.Import
```

Comment:
Can we delete all of `Language.Java.Syntax` while we’re here (especially given its assignment stage is turned off?)

Reply:
Looks like I did, actually?